packages/klevu-ui/src/utils/utils.ts

```diff
@@ -11,8 +20,33 @@ export function renderPrice(amount: number | string, currency: string): string {
   if (window?.["klevu_ui_settings"]?.renderPrice) {
     return window["klevu_ui_settings"].renderPrice(amount, currency)
   }
-  const price = typeof amount === "string" ? parseFloat(amount) : amount
-  return new Intl.NumberFormat(undefined, { style: "currency", currency }).format(price)
+  // Use KMC settings to format price
+  const kmcSettings = getKMCSettings()
```

Comment:
Correct code, but wrong place for it. This should run in `klevu-init` after checking that we are using KMC setting. Original solution should be kept for users who are not using the KMC.